modules/videoio/include/opencv2/videoio/videoio_c.h

```diff
@@ -488,6 +491,13 @@ enum
     CV_CAP_OPENNI_QVGA_60HZ    = 4
 };
 
+// Supported output modes of Aravis image generator
+enum
+{
+    CV_CAP_MODE_GRAY8 = 0,      // Y8 (default)
```

Comment:
@paroj, I have also noticed that CAP_ARAVIS was not defined and CV_CAP_ARAVIS was set to already used enum (CV_CAP_MSMF). Now both of them has fresh & unique ids.

My test code was updated to switch between formats every 5 sec and has been run over the night (for 18h). No memory leaks detected. 

However while switching payload size while high frame rate is set, Aravis looses buffers and no data is received. I treat that as internal AravisSDK problem and shortly will create ticket on that.